addons/l10n_si/data/account_tax_data.xml

```diff
@@ -1,390 +1,1764 @@
-            }),
-        ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'base',
-                'minus_report_line_ids': [ref('tax_report_zo_dvd_odb_vstopni_zni')],
-            }),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_160002'),
-                'minus_report_line_ids': [ref('tax_report_zn_dvd_odb_vstopni_zni')],
-            }),
-            (0,0, {
-                'factor_percent': -100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_260001'),
-                'minus_report_line_ids': [ref('tax_report_zn_dvd_odb_izstopni_zni')],
-            }),
-        ]"/>
-    </record>
-
-    <record id="gd_taxp_st_2" model="account.tax.template">
-        <field name="chart_template_id" ref="gd_chart"/>
-        <field name="name">Vstopni DDV 22% - samoobdavčitev</field>
-        <field name="description">DDV-V-22-SO</field>
-        <field name="amount_type">percent</field>
-        <field name="amount">22</field>
-        <field name="type_tax_use">purchase</field>
-        <field name="price_include" eval="0"/>
-        <field name="tax_group_id" ref="tax_group_0"/>
-        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'base',
-                'plus_report_line_ids': [ref('tax_report_zo_dvd_odb_vstopni_osn')],
-            }),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_160003'),
-                'plus_report_line_ids': [ref('tax_report_zn_dvd_odb_vstopni_osn')],
-            }),
-            (0,0, {
-                'factor_percent': -100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_260002'),
-                'plus_report_line_ids': [ref('tax_report_zn_dvd_odb_izstopni_osn')],
-            }),
-        ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'base',
-                'minus_report_line_ids': [ref('tax_report_zo_dvd_odb_vstopni_osn')],
-            }),
-            (0,0, {
-                'factor_percent': 100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_160003'),
-                'minus_report_line_ids': [ref('tax_report_zn_dvd_odb_vstopni_osn')],
-            }),
-            (0,0, {
-                'factor_percent': -100,
-                'repartition_type': 'tax',
-                'account_id': ref('gd_acc_260002'),
-                'minus_report_line_ids': [ref('tax_report_zn_dvd_odb_izstopni_osn')],
-            }),
-        ]"/>
```

Comment:
Why are we removing the old taxes ? Shouldn't we re-instantiate the new taxes on customer dbs at migration ? (juvr did some helpers for that recently)